server/text_generation_server/models/custom_modeling/flash_deepseek_v2_modeling.py

```diff
@@ -39,6 +39,12 @@
 from transformers.activations import ACT2FN
 from transformers.configuration_utils import PretrainedConfig
 
+if SYSTEM == "rocm":
+    try:
+        from vllm import _custom_C
```

Comment:
Where is this coming from ?

This doesn'tseem to be linked to ruff.

Reply:
this is related to a `undefined-name` lint, `_custom_C` is used in the `DeepseekV2MLP.forward` call but was not imported/defined